src/Tempest/Http/src/Mappers/RequestToObjectMapper.php

```diff
@@ -18,6 +18,8 @@ public function canMap(mixed $from, mixed $to): bool
     public function map(mixed $from, mixed $to): array|object
     {
         /** @var Request $from */
-        return map($from->getBody())->to($to);
+        $data = array_merge($from->getBody(), $from->getFiles());
```

Comment:
Is this the right place though? I thought this mapping should be done in `\Tempest\Http\Mappers\PsrRequestToRequestMapper`